database/insertData/_common/03_template_C_userRegistration.js

```diff
@@ -131,7 +152,254 @@ exports.queries = [
+                      validation: {
+                        operator: "REGEX"
+                        children: [
+                          {
+                            operator: "objectProperties"
+                            children: ["responses.thisResponse"]
+                          }
+                          "^([0]?[1-9]|[1|2][0-9]|[3][0|1])[./-]([0]?[1-9]|[1][0-2])[./-]([0-9]{4}|[0-9]{2})$"
+                        ]
+                      }
+                      validationMessage: "Format expected DD/MM/YYYY"
+                      # TODO: Update to be using a DatePicker element type
+                    }
+                    {
+                      code: "Q7NationalID"
+                      index: 90
+                      title: "National ID number"
+                      elementTypePluginCode: "shortText"
+                      category: QUESTION
+                      parameters: { label: "National ID number" }
+                      validation: {
+                        operator: "REGEX"
+                        children: [
+                          {
+                            operator: "objectProperties"
+                            children: ["responses.thisResponse"]
+                          }
+                          "^[0-9()-]+$"
+                        ]
+                      }
+                      validationMessage: "Must be a number"
+                    }
+                    {
+                      code: "Q8IssuedDate"
+                      index: 100
+                      title: "Date issued"
+                      elementTypePluginCode: "shortText"
+                      category: QUESTION
+                      parameters: { label: "Date issued", maxWidth: 150 }
+                      validation: {
+                        operator: "REGEX"
+                        children: [
+                          {
+                            operator: "objectProperties"
+                            children: ["responses.thisResponse"]
+                          }
+                          "^([0]?[1-9]|[1|2][0-9]|[3][0|1])[./-]([0]?[1-9]|[1][0-2])[./-]([0-9]{4}|[0-9]{2})$"
+                        ]
+                      }
+                      validationMessage: "Format expected DD/MM/YYYY"
+                      # TODO: Update to be using a DatePicker element type
+                    }
+                    {
+                      code: "PB2"
+                      index: 110
+                      title: "Page Break"
+                      elementTypePluginCode: "pageBreak"
+                      category: INFORMATION
+                    }
+                    {
+                      code: "S1BirthAddress"
+                      index: 120
+                      title: "Section 1 - Place of birth"
+                      elementTypePluginCode: "textInfo"
+                      category: INFORMATION
+                      parameters: { title: "## Place of birth", style: "basic" }
```

Comment:
As I mentioned, you can use `style: "none"` (or just leave it out as its the default) to have no "Message box" surround

Reply:
Sure. Good point! Done: 8ca36de